### PR TITLE
fix loading ds2 module issue

### PR DESCRIPTION
**Pull request by wzhongyuan**
## What changes were proposed in this pull request?

Fix rnn loading issue to handle empty (not null) tensor

## How was this patch tested?

Unit test and local test

### CODE REVIEW

**Comment by wzhongyuan** [comment]
PR validation passed
http://172.168.2.101:8080/job/BigDL-PR-V/95/

File: spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/serializer/DataConverter.scala [hunk 1]
```diff
@@ -408,11 +455,15 @@ object DataConverter extends DataConverter{
 
       val storageId = tensor.getTensorType match {
         case DenseType =>
-          System.identityHashCode(tensor.storage().array())
+          if (tensor.isEmpty) -1 else System.identityHashCode(tensor.storage().array())
```
Comment (i8run) [inline spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/serializer/DataConverter.scala]: Why not use `isEmptyTensor`?

Comment (wzhongyuan) [reply spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/serializer/DataConverter.scala]: Refinement, thanks

**Comment by wzhongyuan** [comment]
PR validation passed per review
http://172.168.2.101:8080/job/BigDL-PR-V/101/

**Comment by i8run** [comment]
LGTM

**Pull request merged**